app/services/katello/candlepin/consumer.rb

```diff
@@ -0,0 +1,29 @@
+module Katello
+  module Candlepin
+    class Consumer
+      include LazyAccessor
+
+      attr_accessor :uuid
+
+      def initialize(uuid)
+        self.uuid = uuid
+      end
+
+      def regenerate_identity_certificates
+        Rails.logger.debug "Regenerating consumer identity certificates: #{name}"
+        Resources::Candlepin::Consumer.regenerate_identity_certificates(self.uuid)
+      rescue => e
+        Rails.logger.debug e.backtrace.join("\n\t")
```

Comment:
I'll change to error.  Honestly in most cases we let the traceback bubble up and i think that's probably the right way to handle it.  A lot of this is old code that i am moving around and a lot of the old code had these weird 'catch, log, and re-throw' methods.  My current thinking is to just get rid of the rescue completely.  Thoughts @ehelms ?